Code/Tools/Libs/GuiFoundation/PropertyGrid/Implementation/ElementGroupButton.moc.h

```diff
@@ -13,6 +13,7 @@ class EZ_GUIFOUNDATION_DLL ezQtElementGroupButton : public QToolButton
     MoveElementUp,
     MoveElementDown,
     DeleteElement,
+    Help,
```

Comment:
To make the help button less prominent you can set setAutoRaise(true); for the ezQtElementGroupButton. It might even be a good idea to set that for all types.